### ISSUE TITLE
Code Sandbox and version RC3 don't work together for some strange reason

### ISSUE THREAD

**Issue by brodo** [SAP/ui5-webcomponents#818]
**Describe the bug**
I wanted to file a bug, but cannot set the UI5 version in Code Sandbox to RC3.

**To reproduce**
1. Go to 'https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html'
2. Under 'Dependencies' select 1.0.0-rc.3
3. There is no step 3 

**Screenshots**
![Code Sandbox Problem](https://user-images.githubusercontent.com/377911/66307858-dc33c780-e905-11e9-8872-6ab7388e4866.png)

**Context**
- UI5 Web Components version: 1.0.0-rc.3
- Browser *(if relevant)*: Tested Brave (Chromium) and Firefox Dev edition

I don't know if this should be called a meta-bug or second-order-bug. 🤷‍♂ 😆

**Comment by ilhan007** [comment]
It turns out Code Sandbox fails parsing export statement, if the export statement is not at new line.
We opened an issue for that:
https://github.com/codesandbox/codesandbox-client/issues/2679

```js
// failing in Code Sandbox, although correct syntax
const a = {"a": "value"}; const b = {"b": "value"};export {a, b};

// working
const a = {"a": "value"};
export {
a,
b,
};
```

**Comment by ilhan007** [comment]
We released 1.0.0-rc.4, the Code Sandbox should work fine
